libs/apigateway-lambda-proxy/src/lib/metric-alarms.ts

```diff
@@ -0,0 +1,335 @@
+import * as pulumi from '@pulumi/pulumi'
+import * as aws from '@pulumi/aws'
+
+const namespace = 'AWS/ApiGateway'
+
+/**
+ * Dimensions to identify an HTTP API in CloudWatch.
+ */
+export type HttpGateway = {
+    /**
+     * The API Gateway ID to monitor. Only valid for HTTP endpoints.
+     */
+    id: pulumi.Input<string>
+
+    /**
+     * The stage to monitor
+     */
+    stage?: pulumi.Input<string>
+}
+
+/**
+ * Provides an opinionated set of recommended alarms for an API Gateway stage.
+ *
+ * Raise alarms for the following:
+ *  - 5xx error rate exceeded (default: 2%)
```

Comment:
is 2% high for 5xx? Part of me thinks that it should alarm anything about 0 by default but I'm not sure if that is realistic.

Reply:
It depends on the application under monitoring, for APIs with heavy traffic, it's probably a good default, but for sites with low traffic it's probably too low.